src/ansys/fluent/core/solver/flobject.py

```diff
@@ -292,6 +292,19 @@ class Real(SettingsBase[RealType], Numerical):
     expression values.
     """
 
+    def value(self):
+        """Get the value of the object."""
+        val = self.get_attr("value", (float, int))
```

Comment:
I don't think we should evaluate the `value` attribute directly. `__call__()` or `get_state` will return this?